DevCycle.SDK.Server.Local.MSTests/Fixtures.cs

```diff
@@ -9,10 +12,38 @@ public class Fixtures
         public static string VariationOffId = "6216422850294da359385e90";
         public static string VariableId = "6216422850294da359385e8d";
         public static string VariableKey = "test";
+        public static string LargeConfigVariableKey = "v-key-25";
+        
         public static string Config()
```

Comment:
I will probably go back and replace this with a file as well, just so things are consistent. But i'll do that in a separate PR since I need to touch all the unit tests